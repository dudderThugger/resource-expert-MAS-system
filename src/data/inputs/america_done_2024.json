{"client_id": 0, "base_utterance": null, "plan": [], "research_results": [], "queried_data": {}, "chat_history": [], "current_step": null, "calculation_results": []}