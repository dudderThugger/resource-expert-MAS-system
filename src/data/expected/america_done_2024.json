{"client_id": 0, "base_utterance": null, "plan": [], "research_results": [], "queried_data": {"american_continent_processed_2024": {"Unnamed: 0": {"134": 134, "852": 852, "951": 951, "1140": 1140, "1289": 1289, "1481": 1481, "1679": 1679, "1871": 1871, "2733": 2733, "2917": 2917, "3263": 3263, "3396": 3396, "3445": 3445, "3551": 3551, "3738": 3738, "3975": 3975, "4359": 4359, "4692": 4692, "4697": 4697, "4741": 4741, "4838": 4838, "4886": 4886, "4888": 4888, "4890": 4890, "5272": 5272, "5326": 5326, "5415": 5415, "5418": 5418, "5940": 5940, "6663": 6663, "6710": 6710, "6949": 6949, "6951": 6951, "7237": 7237, "7389": 7389, "7434": 7434, "7582": 7582, "8015": 8015, "8196": 8196, "8205": 8205, "8532": 8532, "8541": 8541, "9451": 9451}, "site_name": {"134": "Piau\u00c3\u00ad_Brazil_160", "852": "Hawaii_United States_235", "951": "Arkansas_United States_214", "1140": "Distrito Federal_Brazil_18", "1289": "Para\u00c3\u00adba_Brazil_76", "1481": "British Columbia_Canada_21", "1679": "New Hampshire_United States_81", "1871": "Nova Scotia_Canada_203", "2733": "South Carolina_United States_1", "2917": "Alberta_Canada_268", "3263": "Alberta_Canada_31", "3396": "Tocantins_Brazil_196", "3445": "Alaska_United States_40", "3551": "Goi\u00c3\u00a1s_Brazil_188", "3738": "Roraima_Brazil_192", "3975": "Tocantins_Brazil_166", "4359": "Rio Grande do Sul_Brazil_160", "4692": "Texas_United States_272", "4697": "Texas_United States_272", "4741": "Manitoba_Canada_297", "4838": "S\u00c3\u00a3o Paulo_Brazil_203", "4886": "Saskatchewan_Canada_290", "4888": "Saskatchewan_Canada_290", "4890": "Saskatchewan_Canada_290", "5272": "Massachusetts_United States_7", "5326": "Colorado_United States_217", "5415": "Mato Grosso_Brazil_236", "5418": "Mato Grosso_Brazil_236", "5940": "Connecticut_United States_111", "6663": "Arkansas_United States_294", "6710": "Par\u00c3\u00a1_Brazil_208", "6949": "Rhode Island_United States_59", "6951": "Rhode Island_United States_59", "7237": "Nunavut_Canada_56", "7389": "Distrito Federal_Brazil_266", "7434": "Ontario_Canada_187", "7582": "New Mexico_United States_99", "8015": "Newfoundland and Labrador_Canada_75", "8196": "Nunavut_Canada_13", "8205": "Nunavut_Canada_13", "8532": "New Jersey_United States_81", "8541": "New Jersey_United States_81", "9451": "Pernambuco_Brazil_279"}, "state": {"134": "Piau\u00c3\u00ad", "852": "Hawaii", "951": "Arkansas", "1140": "Distrito Federal", "1289": "Para\u00c3\u00adba", "1481": "British Columbia", "1679": "New Hampshire", "1871": "Nova Scotia", "2733": "South Carolina", "2917": "Alberta", "3263": "Alberta", "3396": "Tocantins", "3445": "Alaska", "3551": "Goi\u00c3\u00a1s", "3738": "Roraima", "3975": "Tocantins", "4359": "Rio Grande do Sul", "4692": "Texas", "4697": "Texas", "4741": "Manitoba", "4838": "S\u00c3\u00a3o Paulo", "4886": "Saskatchewan", "4888": "Saskatchewan", "4890": "Saskatchewan", "5272": "Massachusetts", "5326": "Colorado", "5415": "Mato Grosso", "5418": "Mato Grosso", "5940": "Connecticut", "6663": "Arkansas", "6710": "Par\u00c3\u00a1", "6949": "Rhode Island", "6951": "Rhode Island", "7237": "Nunavut", "7389": "Distrito Federal", "7434": "Ontario", "7582": "New Mexico", "8015": "Newfoundland and Labrador", "8196": "Nunavut", "8205": "Nunavut", "8532": "New Jersey", "8541": "New Jersey", "9451": "Pernambuco"}, "country": {"134": "Brazil", "852": "United States", "951": "United States", "1140": "Brazil", "1289": "Brazil", "1481": "Canada", "1679": "United States", "1871": "Canada", "2733": "United States", "2917": "Canada", "3263": "Canada", "3396": "Brazil", "3445": "United States", "3551": "Brazil", "3738": "Brazil", "3975": "Brazil", "4359": "Brazil", "4692": "United States", "4697": "United States", "4741": "Canada", "4838": "Brazil", "4886": "Canada", "4888": "Canada", "4890": "Canada", "5272": "United States", "5326": "United States", "5415": "Brazil", "5418": "Brazil", "5940": "United States", "6663": "United States", "6710": "Brazil", "6949": "United States", "6951": "United States", "7237": "Canada", "7389": "Brazil", "7434": "Canada", "7582": "United States", "8015": "Canada", "8196": "Canada", "8205": "Canada", "8532": "United States", "8541": "United States", "9451": "Brazil"}, "service_month": {"134": "MAR-2024", "852": "JAN-2024", "951": "APR-2024", "1140": "JAN-2024", "1289": "JUN-2024", "1481": "JUN-2024", "1679": "DEC-2024", "1871": "DEC-2024", "2733": "OCT-2024", "2917": "FEB-2024", "3263": "DEC-2024", "3396": "JAN-2024", "3445": "FEB-2024", "3551": "DEC-2024", "3738": "JUL-2024", "3975": "APR-2024", "4359": "APR-2024", "4692": "JAN-2024", "4697": "JUN-2024", "4741": "FEB-2024", "4838": "MAR-2024", "4886": "MAR-2024", "4888": "MAY-2024", "4890": "JUL-2024", "5272": "MAY-2024", "5326": "NOV-2024", "5415": "APR-2024", "5418": "JUL-2024", "5940": "JAN-2024", "6663": "APR-2024", "6710": "MAR-2024", "6949": "FEB-2024", "6951": "APR-2024", "7237": "FEB-2024", "7389": "OCT-2024", "7434": "JUL-2024", "7582": "NOV-2024", "8015": "DEC-2024", "8196": "JAN-2024", "8205": "OCT-2024", "8532": "JAN-2024", "8541": "OCT-2024", "9451": "AUG-2024"}, "invoice_name": {"134": "Piau\u00c3\u00ad_Brazil_160-MAR-2024", "852": "Hawaii_United States_235-JAN-2024", "951": "Arkansas_United States_214-APR-2024", "1140": "Distrito Federal_Brazil_18-JAN-2024", "1289": "Para\u00c3\u00adba_Brazil_76-JUN-2024", "1481": "British Columbia_Canada_21-JUN-2024", "1679": "New Hampshire_United States_81-DEC-2024", "1871": "Nova Scotia_Canada_203-DEC-2024", "2733": "South Carolina_United States_1-OCT-2024", "2917": "Alberta_Canada_268-FEB-2024", "3263": "Alberta_Canada_31-DEC-2024", "3396": "Tocantins_Brazil_196-JAN-2024", "3445": "Alaska_United States_40-FEB-2024", "3551": "Goi\u00c3\u00a1s_Brazil_188-DEC-2024", "3738": "Roraima_Brazil_192-JUL-2024", "3975": "Tocantins_Brazil_166-APR-2024", "4359": "Rio Grande do Sul_Brazil_160-APR-2024", "4692": "Texas_United States_272-JAN-2024", "4697": "Texas_United States_272-JUN-2024", "4741": "Manitoba_Canada_297-FEB-2024", "4838": "S\u00c3\u00a3o Paulo_Brazil_203-MAR-2024", "4886": "Saskatchewan_Canada_290-MAR-2024", "4888": "Saskatchewan_Canada_290-MAY-2024", "4890": "Saskatchewan_Canada_290-JUL-2024", "5272": "Massachusetts_United States_7-MAY-2024", "5326": "Colorado_United States_217-NOV-2024", "5415": "Mato Grosso_Brazil_236-APR-2024", "5418": "Mato Grosso_Brazil_236-JUL-2024", "5940": "Connecticut_United States_111-JAN-2024", "6663": "Arkansas_United States_294-APR-2024", "6710": "Par\u00c3\u00a1_Brazil_208-MAR-2024", "6949": "Rhode Island_United States_59-FEB-2024", "6951": "Rhode Island_United States_59-APR-2024", "7237": "Nunavut_Canada_56-FEB-2024", "7389": "Distrito Federal_Brazil_266-OCT-2024", "7434": "Ontario_Canada_187-JUL-2024", "7582": "New Mexico_United States_99-NOV-2024", "8015": "Newfoundland and Labrador_Canada_75-DEC-2024", "8196": "Nunavut_Canada_13-JAN-2024", "8205": "Nunavut_Canada_13-OCT-2024", "8532": "New Jersey_United States_81-JAN-2024", "8541": "New Jersey_United States_81-OCT-2024", "9451": "Pernambuco_Brazil_279-AUG-2024"}, "submitted_by": {"134": "Jeff W.", "852": "Eszter S.", "951": "Eszter S.", "1140": "Jeff W.", "1289": "Carlos R.", "1481": "Carlos R.", "1679": "Jeff W.", "1871": "Carlos R.", "2733": "Adam S.", "2917": "Jeff W.", "3263": "Jeff W.", "3396": "Adam S.", "3445": "Jeff W.", "3551": "Carlos R.", "3738": "Carlos R.", "3975": "Jeff W.", "4359": "Carlos R.", "4692": "Jeff W.", "4697": "Jeff W.", "4741": "Eszter S.", "4838": "Carlos R.", "4886": "Jeff W.", "4888": "Adam S.", "4890": "Jeff W.", "5272": "Carlos R.", "5326": "Adam S.", "5415": "Carlos R.", "5418": "Adam S.", "5940": "Eszter S.", "6663": "Jeff W.", "6710": "Eszter S.", "6949": "Eszter S.", "6951": "Adam S.", "7237": "Carlos R.", "7389": "Eszter S.", "7434": "Adam S.", "7582": "Adam S.", "8015": "Jeff W.", "8196": "Carlos R.", "8205": "Jeff W.", "8532": "Eszter S.", "8541": "Eszter S.", "9451": "Carlos R."}, "status": {"134": "PROCESSED", "852": "PROCESSED", "951": "PROCESSED", "1140": "PROCESSED", "1289": "PROCESSED", "1481": "PROCESSED", "1679": "PROCESSED", "1871": "PROCESSED", "2733": "PROCESSED", "2917": "PROCESSED", "3263": "PROCESSED", "3396": "PROCESSED", "3445": "PROCESSED", "3551": "PROCESSED", "3738": "PROCESSED", "3975": "PROCESSED", "4359": "PROCESSED", "4692": "PROCESSED", "4697": "PROCESSED", "4741": "PROCESSED", "4838": "PROCESSED", "4886": "PROCESSED", "4888": "PROCESSED", "4890": "PROCESSED", "5272": "PROCESSED", "5326": "PROCESSED", "5415": "PROCESSED", "5418": "PROCESSED", "5940": "PROCESSED", "6663": "PROCESSED", "6710": "PROCESSED", "6949": "PROCESSED", "6951": "PROCESSED", "7237": "PROCESSED", "7389": "PROCESSED", "7434": "PROCESSED", "7582": "PROCESSED", "8015": "PROCESSED", "8196": "PROCESSED", "8205": "PROCESSED", "8532": "PROCESSED", "8541": "PROCESSED", "9451": "PROCESSED"}}}, "chat_history": [], "current_step": null, "calculation_results": []}